src/components/ContinuationIn/ExtraproBusinessLookup.vue

```diff
@@ -34,9 +34,8 @@
             cols="3"
             class="result-identifier d-inline-flex"
           >
-            {{ item.identifier }}
-            <!-- *** TODO: remove after testing: -->
-            ({{ item.status?.charAt(0) }})
+            <span>{{ item.identifier }}</span>
+            <span v-if="showBusinessSearchStatus">&nbsp;({{ item.status?.charAt(0) }})</span>
```

Comment:
I kept the Active/Historical indicator but feature-flagged it. It's super useful for testing but we don't want to show it in Prod.